packit/upstream.py

```diff
@@ -152,7 +152,9 @@ def get_latest_released_version(self) -> Optional[str]:
         """
 
         version = get_upstream_version(self.package_config.downstream_package_name)
-        logger.info(f"Version in upstream registries is {version!r}.")
+        logger.info(
+            f"Version in release-monitoring.org is {version!r}.",
```

Comment:
I don't know, `in` doesn't sound right, but `at` is a bit weird as well, what about this?

```suggestion
            f"Version retrieved from release-monitoring.org is {version!r}.",
```